components/feature/syncedtabs/src/main/java/mozilla/components/feature/syncedtabs/controller/DefaultController.kt

```diff
@@ -57,6 +55,7 @@ internal class DefaultController(
      * See [SyncedTabsController.syncAccount]
      */
     override fun syncAccount() {
+        view.startLoading()
```

Comment:
Without symmetrical calls with start/stop, couldn't we get into a state where we tell the view we are forever loading and never reach the view.stopLoading call? i.e. `onIdle` is not called.

I don't have any solutions for this, but it's mostly a note.

Reply:
Yeah, I think there is a risk of that for sure. It's a side-effect of the manually registered observer paradigm that this is still using. I'm not sure there's a great way to avoid it, currently, but it should be dealt with during the creation of the `SyncStore`. 

Are you okay with landing this change as is for now? We may even get to the `SyncStore` before my work with the recent synced tab ships, as I think we're targeting the MR release for it. Still some risk with the synced tabs tray, but I think that's the only other spot using this code.

Reply:
It's a concern to keep an eye out, but let's land it (after the beta cut). So we can see how it goes in nightly.